gnosis_vpn-lib/src/core/disconn.rs

```diff
@@ -0,0 +1,84 @@
+use serde::{Deserialize, Serialize};
+
+use std::fmt::{self, Display};
+use std::time::SystemTime;
+
+use crate::connection::destination::Destination;
+use crate::core::conn::Conn;
+use crate::core::disconnection_runner;
+use crate::{log_output, wg_tooling};
+
+#[derive(Clone, Debug)]
+pub struct Disconn {
```

Comment:
The letters are not expensive, why not:

```suggestion
pub struct Disconnect {
```